File: phases/00-setup-and-tooling/04-apis-and-keys/code/apis.py
import os
from dotenv import load_dotenv    
import anthropic
load_dotenv()
client = anthropic.Anthropic(
    api_key=os.getenv("ANTHROPIC_API_KEY"),
    base_url="https://api.deepseek.com/anthropic"
)

response = client.messages.create(
    model="deepseek-v4-pro",
    max_tokens=1000,
    messages=[{"role": "user", 
               
               "content": [
                {
                    "type": "text",
                    "text": "What is a neural network in one sentence?"
                }
            ]
               
               }
               
               ]
)

for block in response.content:
    if block.type == "text":
        print(block.text)

File: phases/00-setup-and-tooling/04-apis-and-keys/code/apis.ts
import "dotenv/config";
import Anthropic from "@anthropic-ai/sdk";

const client = new Anthropic({
    apiKey: process.env.ANTHROPIC_API_KEY,
    baseURL: "https://api.deepseek.com/anthropic"
});

const response = await client.messages.create({
    model: "deepseek-v4-flash",
    max_tokens: 10000,
    messages: [
        {
            role: "user",
            content: "What is the capital of France?"
        }
    ]
});

// 只打印 text 类型的块的内容
response.content.forEach(block => {
    if (block.type === 'text') console.log(block.text);
});